app/frontend/src/components/Inputs/Calendar.tsx

```diff
@@ -145,55 +145,58 @@ function CalendarRow(props: CalendarRowProps) {
   );
 }
 
-function CalendarCell({ selected = false, ...props }: CalendarCellProps) {
-  const [isSelected, setIsSelected] = useState(selected);
+function CalendarCell({
+  cell,
+  setToSelect,
+  setIsMouseDown,
+  isMouseDown,
+  data,
+  setData,
+  toSelect,
+}: CalendarCellProps) {
+  const [next, nextdata, handleSelect] = useDragToSelectUnselect(
+    { ...cell },
```

Comment:
I am not sure this destructuring does anything since it just recreates the cell object.

Reply:
It doesn't. I was testing something before and forgot to undo the change. Thanks for flagging.